### ISSUE TITLE
Consistently denoting identity matrix.

### ISSUE THREAD

**Issue by brenjohn** [shinuito/mt543_notes#17]
I think at some point I started using $\cin$ to denote the identity matrix when other notation was already being used. It would be good to have consistent notation throughout the notes.

**Comment by shinuito** [comment]
Yeah it makes sense - I think I was keeping `\cin` for a fairly specific thing early on to do with complex matrices (I think) - I'll have to double check later.
